src/schema/validator.js

```diff
@@ -542,6 +542,67 @@ export class SchemaValidator {
       ),
       errors: true,
     });
+
+    const getPrivilegedPermissionsSet =
+      this.getPrivilegedPermissionsSet.bind(this);
+
+    validator.addKeyword({
+      keyword: 'postprocess',
+      validate: function validate(keywordSchemaValue, propValue) {
+        if (keywordSchemaValue === 'checkPrivilegePermission') {
+          const privilegedPermissions = getPrivilegedPermissionsSet(validator);
```

Comment:
Is this needed to have the right `this` in the validate function below? Maybe try arrow function for the `validate`.